web/modules/custom/modal_form_example/src/Form/Demo.php

```diff
@@ -0,0 +1,314 @@
+                <i class="fas fa-angle-down"></i>
+                </div>',
+      ];
+      $form['accessible'] = [
+        '#type' => 'textfield',
+        '#prefix' =>
+        '<div class = "accessibility" id ="accessibility">
+                    <div class = "access1" id="access1">
+                    <h2>Accessibility</h2>
+                   <i class="fas fa-angle-up"></i>
+                    </div>
+                    <p>Alt text describes your photos for people with visual impairments. Alt text will be automatically created for your photos or you can choose to write your own.</p>
+                    <div class = "alttext"><img src = "/sites/default/files/' .
+        $uri .
+        '"> ',
+        '#placeholder' => ' Write alt Text',
+        '#suffix' => '</div>',
+        '#suffix' => '</div>',
+        '#suffix' => '</div>',
+
+      ];
+    }
+
+    if ($this->step == 1) {
+      $form['buttons']['forward'] = [
+        '#type' => 'submit',
+        '#value' => $this->t('Next'),
+
+        '#attributes' => [
+          'class' => ['nextbutton'],
+        ],
+        '#ajax' => [
+                // We pass in the wrapper we created at the start of the form.
+          'wrapper' => 'ajax_form_multistep_form',
+                // We pass a callback function we will use later
+                // to render the form for the user.
+          'callback' => '::ajaxFormMultistepFormAjaxCallback',
+          'event' => 'click',
+        ],
+      ];
+    }
+    if ($this->step == 2) {
+      $form['buttons']['forward'] = [
+        '#type' => 'submit',
+        '#prefix' => '</div>',
+        '#value' => $this->t('Share'),
+        '#attributes' => [
+          'class' => ['submitbutton'],
+        ],
+        '#ajax' => [
+                // We pass in the wrapper we created at the start of the form.
+          'wrapper' => 'ajax_form_multistep_form',
+                // We pass a callback function we will
+                // use later to render the form for the user.
+          'callback' => '::ajaxFormMultistepFormAjaxCallback',
+          'event' => 'click',
+        ],
+      ];
+      $form['backword'] = [
+        '#type' => 'submit',
+        '#value' => '<-',
+        '#attributes' => [
+          'class' => ['backbutton'],
+        ],
+        '#ajax' => [
+            // We pass in the wrapper we created at the start of the form.
+          'wrapper' => 'ajax_form_multistep_form',
+                  // We pass a callback function we will use later
+                  // to render the form for the user.
+          'callback' => '::ajax_form_back_ajax_callback',
+          'event' => 'click',
+        ],
+      ];
+    }
+    if ($this->step == 3) {
+      $form['message-title'] = [
+        '#markup' =>
+        '<div class ="complete"> <img src = "/modules/custom/modal_form_example/inline-images/10a8cbeb94ba.gif"></i> <h2>'
+        . $this->t('Your post has been shared') .
+        '</h2></div>',
+      ];
```

Comment:
use render array here, Also module path can change, use `drupal_get_path($module)` some function like this to get the module path.